crates/gitbutler-tauri/Cargo.toml

```diff
@@ -69,19 +77,10 @@ gitbutler-edit-mode.workspace = true
 open = "5"
 
 [dependencies.tauri]
-version = "1.7.0"
+version = "2.0.0-rc"
```

Comment:
Looks like this should be `v2.0.0-rc.10`.

Reply:
I believe it's a valid semver

Reply:
It is, but atm we prefer pinned versions.

Reply:
I think it should be `=2.0.0-rc.10` right?

Reply:
Are you saying the rc versions are treated as patches and therefore need the added equal sign to force exact matching?

Reply:
yes, otherwise it will include `2.0.0-rc.11 ` and even `2.0.0`